conf/base.config

```
@@ -133,4 +133,9 @@ process {
         // 100k reads per hour for PacBio, 50m for HiC
         time   = { 1.h  * Math.ceil( meta.read_count / (meta.datatype == "pacbio" ? 100000 : 50000000)) * task.attempt }
     }
+
+    withName: 'BLOBTK_DEPTH|BGZIP_BEDGRAPH' {
+        memory = { 1.GB  * Math.ceil( meta.read_count / 1000000 ) * task.attempt }
```

I don't know how much memory they use exactly, but I think BLOBTK_DEPTH depends more on the genome size than the number of reads, and BGZIP is just compressing the file by blocks so its memory usage should be constant.
In any case, I think we have sequencing runs with more than a billion reads, so let's just remove the dependency on `meta.read_count`. We'll have a first taste of real world usage through the full test.

```suggestion
        memory = { 4.GB * task.attempt }
```